From: Ben Hutchings <ben@decadent.org.uk>
Date: Mon, 27 Oct 2014 00:17:13 +0000
Subject: drivers/net: Avoid ABI change for UFO/IPv6 fix
Forwarded: not-needed

Hide new header inclusion from genksyms.

The functions exported from these drivers (macvtap_get_socket() and
tun_get_socket()) don't seem to be used by anything in wheezy but
it looks like they could be used by a backport of vhost.

--- a/drivers/net/macvtap.c
+++ b/drivers/net/macvtap.c
@@ -15,7 +15,9 @@
 #include <linux/cdev.h>
 #include <linux/fs.h>
 
+#ifndef __GENKSYMS__
 #include <net/ipv6.h>
+#endif
 #include <net/net_namespace.h>
 #include <net/rtnetlink.h>
 #include <net/sock.h>
--- a/drivers/net/tun.c
+++ b/drivers/net/tun.c
@@ -64,7 +64,9 @@
 #include <linux/nsproxy.h>
 #include <linux/virtio_net.h>
 #include <linux/rcupdate.h>
+#ifndef __GENKSYMS__
 #include <net/ipv6.h>
+#endif
 #include <net/net_namespace.h>
 #include <net/netns/generic.h>
 #include <net/rtnetlink.h>
